pkgs/development/python-modules/ywsd/default.nix

```diff
@@ -0,0 +1,53 @@
+{ lib
+, aiopg
+, aiohttp
+, buildPythonPackage
+, fetchFromGitHub
+, python-yate
+, pyyaml
+, sqlalchemy
+}:
+
+buildPythonPackage rec {
+  pname = "ywsd";
+  version = "0.13.1";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "eventphone";
+    repo = "ywsd";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-eNDRhnXQYnlxRJ5xfJ6nlYALoJd2ecdLmRTwBS2/H6A=";
+  };
+
+  propagatedBuildInputs = [
+    aiopg
+    aiohttp
+    python-yate
+    pyyaml
+    (sqlalchemy.overridePythonAttrs (oldAttrs: rec {
+      version = "1.4.49";
+      src = fetchFromGitHub {
+        owner = "sqlalchemy";
+        repo = "sqlalchemy";
+        rev = "refs/tags/rel_${lib.replaceStrings [ "." ] [ "_" ] version}";
+        hash = "sha256-v7xHfN8we7BZ1eQwrQ/jMnlWgxsmrdYPsHpAFeS9Fhg=";
+      };
+      doCheck = false;
+    }))
```

Comment:
can you add a comment explaining why you are overriding this here?

Reply:
We cannot override other packages in python modules. This is because the dependencies become too complex to control.
Is this an application? If so, please move it to pkgs/by-name/... instead of python modules.
You can leave the overrides as they are.

Reply:
Is this an application is a good question. At least I handle it as a module in practice. I load the database models and start it in another thread. Is there any other option for handling the dependency?

Reply:
> Is there any other option for handling the dependency?

Unfortunately, no.
We can't add it as a library to nixpkgs unless upstream supports the current sqlalchemy.
You can have personal flakes or overlays.

Reply:
Okay thank you. I'm just going to leave it here now and we can try again when upstream fixes it.